packages/components/spacings/spacings-inline/src/types.ts

```diff
@@ -0,0 +1,27 @@
+// @TODO extract this into a shared file
+export type TAlignItem =
+  | 'stretch'
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'baseline'
+  // Deprecated
+  | 'flexStart'
+  | 'flexEnd';
+
+export type TJustifyContent =
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evenly';
+
+export type TScale = 'xs' | 's' | 'm' | 'l' | 'xl';
+
+export type TProps = {
+  scale?: TScale;
+  alignItems?: TAlignItem;
+  justifyContent?: TJustifyContent;
+  children?: React.ReactNode;
```

Comment:
Strange that children are optional but seem sit was in the prop-types too.

Reply:
It somehow makes sense, if we want to be flexible in nullified node instance

```jsx
<Spacings.Stack>{isAuthorized ? <div />: null }</Spacings.Stack>
```

Reply:
Fair point. Just wondered if then `<Spacings.Stack>{isAuthorized ? <div />: </> }</Spacings.Stack>` would be more apt. I don't know if fragment counted as a React.ReactNode I just can't remember React's type inheritance.

Anyway, thanks for clarifying.

Reply:
`null` should be a valid `ReactNode` actually. I would keep `children` as required.